cypress/support/fragments/circulation/notice-policy-template.js

```diff
@@ -0,0 +1,52 @@
+import uuid from 'uuid';
+import getRandomPostfix from '../../utils/stringTools';
+
+const defaultNoticeTemplateBody = {
+  active: true,
+  description: 'Notice_policy_template_description',
+  id: uuid(),
+  localizedTemplates: {
+    en: { header: 'Email_subject', body: '<div>Email_body</div>' },
+  },
+  body: '<div> Email_body</div>',
+  header: 'Ema il_subject',
+  name: `Tempp late_name_${getRandomPostfix}`,
+  outputFormats: ['text/html'],
+  0: 'text/html',
+  templateResolver: 'mustache',
+};
+
+export const TEMPLATE_CATEGORIES = {
+  loan: 'Loan',
+  request: 'Request',
+  AutomatedFeeFineCharge: 'Automated fee/fine charge',
+  AutomatedFeeFineAdjustment: 'Automated fee/fine adjustment (refund or cancel)',
+  FeeFineCharge: 'Manual fee/fine charge',
+  FeeFineAction: 'Manual fee/fine action (pay, waive, refund, transfer or cancel/error)'
+};
+
+export default {
+  defaultNoticeTemplateBody,
+  createViaApi(noticeTemplateCategory) {
+    return cy.okapiRequest({ method: 'POST',
+      path: 'templates',
+      body: {
+        category: noticeTemplateCategory,
```

Comment:
I suggest change order. Initially it should be ...defaultNoticeTemplateBody. After this category: noticeTemplateCategory,.

Reply:
i moved in the notice-policy file[ ](https://github.com/folio-org/stripes-testing/blob/18d3650fd52822371eb2fde30f55a72f1c07a40d/cypress/support/fragments/circulation/notice-policy.js#L9)